File: CourseCodesAPI/CourseCodesAPI/Models/AccountCreateRequest.cs
using CourseCodesAPI.Entities;

namespace CourseCodesAPI.Models
{
	public class AccountCreateRequest
	{
		public string FirstName { get; set; }
		public string LastName { get; set; }
		public string Email { get; set; }
		public string PasswordHash { get; set; }
		public Role AccountRole { get; set; }
	}
}

File: coursecodesapp/src/api/models/AccountCreateRequest.js
import { ApiHelpers } from '../ApiHelpers';

export class AccountCreateRequest {
	constructor({ firstName, lastName, email, password, accountRole }) {
		this.firstName = firstName;
		this.lastName = lastName;
		this.email = email;
		this.passwordHash = ApiHelpers.toHash(password);
		this.accountRole = accountRole;
	}
}
